.github/workflows/build-test.yml

```diff
@@ -646,17 +635,6 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: echo "xvfbCommand=--server-args=\"-screen 0 1024x768x24\"" >> $GITHUB_ENV
 
-      - name: Run tests with VSCode & Jupyter
-        uses: GabrielBB/xvfb-action@v1.4
-        with:
-          run: ${{ env.xvfbCommand }} npm run testVSCode
```

Comment:
This is a subset of the notebook tests (see comments in the PR).
Hence one less job, fewer resources